ffv1.md

```diff
@@ -233,15 +233,26 @@ RFC:```
 
 If the context is smaller than 0 then -context is used and the difference between the sample and its predicted value is encoded with a flipped sign.
 
-## Quantization
+## Quantization tables
 
-There are 5 quantization tables for the 5 sample differences, both the number of quantization steps and their distribution are stored in the bitstream. Each quantization table has exactly 256 entries, and the 8 least significant bits of the sample difference are used as index:
+Each Quantization Table contains 5 exactly Quantization Table Blocks, each block corresponding to 1 of the 5 Quantized Sample Differences, both the number of quantization steps and their distribution [are stored in the bitstream](#quantization-table). Each Quantization Table Block has exactly 256 entries, and the 8 least significant bits of the Quantized Sample Differences are used as index:
 
-PDF:$$Q_{i}[a-b]=Table_{i}[(a-b)\&255]$$
+PDF:$$Q_{j}[k]=quant_tables[i][j][k\&255]$$
 RFC:```
-RFC:Q_{i}[a − b] = Table_{i}[(a − b)&255]
+RFC:Q_{j}[k] = quant_tables[i][j][k&255]
 RFC:```
 
+In this formula, `i` is the Quantization Table index of the currently parsed data, `j` is the Quantized Sample Differences index, `k` the Quantized Sample Differences.
+
+## Quantization Table indexes
+
+For each plane of each slice, a Quantization Table is selected from an index:
```

Comment:
You may need an extra line break after line 249 for the list to render correctly in the RFC outputs.